vmdb/app/views/container_node/_main.html.haml

```diff
@@ -3,3 +3,4 @@
   .col-sm-12.col-md-12.col-lg-6
     = render :partial => "shared/summary/textual", :locals => { :title => _("Properties"), :items => textual_group_properties }
     = render :partial => "shared/summary/textual", :locals => { :title => _("Relationships"), :items => textual_group_relationships }
+    = render :partial => "shared/summary/textual_node_conditions", :locals => { :title => _("Conditions"), :items => textual_group_conditions, :labels => [_("Name"), _("Status"), _("Last Transition Time"), _("Reason")]}
```

Comment:
@dkorn can we rename this partial to be generic so it can be used on other summary views in future. Then dont need to pass in labels from main.html.haml instead change def textual_group_conditions method to pass in values and labels like this:
def textual_group_conditions
    labels = [_("Name"), _("Status"), _("Last Transition Time"), _("Reason")]
    h = {:labels => labels}
    h[:values] = @record.container_node_conditions.collect do |condition|
                  [
                    condition.name,
                    condition.status,
                    (condition.last_transition_time || "<None>"),
                    (condition.reason || "<None>")
                  ]
                end
    h
  end

Once shared/summary/textual_node_conditions file is renamed that can use labels and values that are being returned from  textual_group_conditions method
- if items
  %table.table.table-bordered.table-striped
    %thead
      %tr
        %th{:colspan => 6, :align => "left"}
          = title
    - items[:labels].each do |label|
      %td
        %strong
          = label
    - items[:values].each do |item|
      %tr
        - item.each do |value|
          %td
            = value

let me know if you have questions.